tests/installation/addon_products_via_SCC_yast2.pm

```diff
@@ -26,19 +26,20 @@ sub run() {
             push(@addon_proxy, "\b.$addon-" . get_var("BUILD_$uc_addon"));
         }
         script_sudo "echo \"@addon_proxy.proxy.scc.suse.de\" > /etc/SUSEConnect", 0;
+        script_sudo 'zypper mr -d 1',                                             0;    # zypper_lr test is expecting this repo disabled
```

Comment:
can you guarantee that repo #1 is ALWAYS going to be the one you want to disable? Seeing a magic number ilke that makes me nervous